src/DataFrames.jl

```diff
@@ -63,8 +63,10 @@ export AbstractDataFrame,
 
 if VERSION >= v"1.1.0-DEV.792"
     import Base.eachcol, Base.eachrow
+    export select!, select
```

Comment:
This is redundant with the export above.

Reply:
doh, fixed, thanks.